src/lib/structures/MusicManager.js

```diff
@@ -232,16 +233,13 @@ class MusicManager {
 
 		const [song] = this.queue;
 
-		const stream = ytdl(`https://youtu.be/${song.url}`, {
-			filter: song.opus
-				? format => format.type === 'audio/webm; codecs="opus"'
-				: 'audioonly'
-		}).on('error', err => this.client.emit('error', err));
+		const stream = await ytdlDiscord(`https://youtu.be/${song.url}`)
+			.catch(err => this.client.emit('error', err));
```

Comment:
Nuke this `catch`, if it fails, stream will be `undefined`, causing a double error (the second being misleading).